apps/new-dealer/controllers/supporting-documents.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const BaseController = require('../../common/controllers/base');
+const UploadModel = require('../../common/models/file-upload');
+
+const uuid = require('node-uuid');
```

Comment:
use `uuid`, a fork of node-uuid, maintained more regularly and https://github.com/kelektiv/node-uuid/issues/116